modules/microshift-manifests-deletion-overview.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * microshift//running_applications/microshift-deleting-resource-manifests.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="microshift-manifests-deletion-overview_{context}"]
+= How manifest deletion works
+
+By default, {microshift-short} expects to find deletion manifests in the `delete` subdirectories within the manifests path. When a manifest is placed in these subdirectories, {microshift-short} removes the manifests when the system is started. Read through the following to understand how manifests deletion works in {microshift-short}.
+
+. At every start, before applying the manifests, {microshift-short} scans the following `delete` sub directories within the configured manifests directory to identify the Kustomize manifests that need to be deleted:
```

Comment:
I agree that "start" reads awkward for some reason. Is this right? I think it agrees with the intro paragraph.
```suggestion
. Each time the system starts, before applying the manifests, {microshift-short} scans the following `delete` subdirectories within the configured manifests directory to identify the Kustomize manifests that need to be deleted:
```

Reply:
done